File: PHP Excel Examples/CellsApiDemo/Cell/Merge/unmergeCells.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "UnMergeCells.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet2";
$startRow = 3;
$startColumn = 1;
$totalRows = 1;
$totalColumns = 2;
$apiInstance->unmergeCells($name, $sheetName, $startRow, $startColumn, $totalRows, $totalColumns, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/cell/unMergeCells.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "UnMergeCells_1.xlsx"
folder = "/Cells/Cell/Merge/"
storage = ""
sheetName = "Sheet2"
startRow = 3
startColumn = 1
totalRows = 1
totalColumns = 2
api.unmerge_cells(name, sheetName, startRow, startColumn, totalRows, totalColumns, folder=folder, storage=storage)
